### ISSUE TITLE
"No error message" on Vagrant errors

### ISSUE THREAD

**Issue by Temikus** [mitchellh/vagrant-google#45]
Currently, if we define an external IP for the instance that is already taken we will get an error message, however, the follow-up text in red says `No error message` which seems to be incorrect:

```
==> default: Specified external_ip is already in use, cannot be used!
No error message
```

**Comment by Temikus** [comment]
Introduced in b4817f87a5df6af65d1a652c0f2d9f51050a5bcf and behaviour foolishly copied by me in #38.
Error class should either provide an error key in vagrant.errors namespace for i18n to pick up *or* define a new exception as raise MyError.new, not raise a generic Errors::VagrantGoogleError

/CC @erjohnso 
Can be closed once #47 merges.

**Comment by erjohnso** [comment]
#47 merged - closing. Thanks @Temikus !
